src/app/tool-editor/tool-editor.component.ts

```diff
@@ -282,6 +287,27 @@ export class ToolEditorComponent extends DirectiveBase implements OnInit, OnDest
         });
     }
 
+    hasCopyOfProperty() {
+        return typeof this.data.parsedContent["sbg:copyOf"] !== undefined;
```

Comment:
Check that it isn't a local app, in case the user copied a platform app to their desktop.